engines/scalac/src/main/scala/scala/macros/internal/engines/scalac/trees/Abstracts.scala

```diff
@@ -192,7 +193,9 @@ trait Abstracts extends scala.macros.trees.Abstracts with Positions { self: Univ
           if (targs.nonEmpty) method = g.TypeApply(method, targs).setPos(op.pos)
           g.Apply(method, args)
         } else {
-          ???
+          var method: g.Tree = g.Select(args.head, op.toGTermName).setPos(op.pos)
+          if (targs.nonEmpty) method = g.TypeApply(method, targs).setPos(op.pos)
+          g.Apply(method, lhs :: Nil)
```

Comment:
I assume yes. Just create right associative expression and compile with key -print and printTree or smth like that. I'll try it